action/protocol/rewarding/fund.go

```diff
@@ -146,3 +167,35 @@ func DepositGas(ctx context.Context, sm protocol.StateManager, amount *big.Int)
 	}
 	return rp.Deposit(ctx, sm, amount, iotextypes.TransactionLogType_GAS_FEE)
```

Comment:
rp.deposit()

Reply:
inside `rp.Deposit()`, it will fallback to `deposit()`
`rp.Deposit()` is also used somewhere else, so just want to keep it the same